gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
```

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```

all or bin? All has sources that we probably don't need?

---

bin. Thanks for catching this, I forgot to change it after checking some info. 😅